src/classes/Countertop.js

```diff
@@ -13,18 +14,26 @@ import CountertopTopology from './CountertopTopology'
 class Countertop {
 	logger = null
 
+	kafka = null
+
 	stations = []
 
 	state = ''
 
 	/**
-	 * @param  {Logger} options.logger A logger with methods for all TV Kitchen logLevels.
+	 * @param  {Logger} options.logger        A logger with methods for all TV Kitchen logLevels.
+	 * @param  {Object} options.kafkaSettings Kafka settings as defined in the kafkajs library.
 	 */
 	constructor({
 		logger = consoleLogger,
+		kafkaSettings = {
+			clientId: 'tv-kitchen',
+			brokers: ['127.0.0.1:9092'],
```

Comment:
I'm not sure we should actually provide any default values at all, for anything here.

Reply:
I'm going to take out this default for now.  We can add one some day if we want to.